src/components/import/Import.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Breadcrumbs,
+  Card,
+  CardContent,
+  Container,
+  Link,
+  Typography
+} from '@mui/material'
+import DropZone from './DropZone'
+
+const Import = () => {
+  return (
+    <Container>
+      <Breadcrumbs>
+        <Link underline="hover" color="inherit" href="/">
+          Home
+        </Link>
+        <Typography color="text.primary">Import</Typography>
+      </Breadcrumbs>
+      <Typography color="text.primary" variant="h4">
+        Import
+      </Typography>
```

Comment:
You could re-use the PageHeader component to display the title and breadscrumbs.